### PR TITLE
Current number of votes

### PR DESCRIPTION
**Pull request by jclariviere**
Security will be done in another PR

### CODE REVIEW

File: src/main/java/com/arcbees/bourseje/client/model/Candidate.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ *  * Copyright 2015 ArcBees Inc.
```
Comment (christiangoudreau) [inline src/main/java/com/arcbees/bourseje/client/model/Candidate.java]: seems copy pasted :D

**Comment by christiangoudreau** [comment]
LGTM

File: src/main/java/com/arcbees/bourseje/client/model/Candidate.java (hunk 1 above)
Comment (jclariviere) [inline src/main/java/com/arcbees/bourseje/client/model/Candidate.java]: Oops!

Comment (jclariviere) [inline src/main/java/com/arcbees/bourseje/client/model/Candidate.java]: It was a problem with Intellij.

File: src/main/java/com/arcbees/bourseje/server/services/VoteService.java [hunk 2]
```diff
@@ -76,6 +82,32 @@ public void vote(VoteItem voteItem, String code) {
         voteItemDao.put(voteItem);
     }
 
+    public Collection<CandidateResult> getVotesPerCandidate() {
+        List<VoteItem> allVotes = voteItemDao.getAll();
```
Comment (Chris-V) [inline src/main/java/com/arcbees/bourseje/server/services/VoteService.java]: So you go all in with eventual consistency? :p

**Comment by Chris-V** [comment]
LGTM.

File: src/main/java/com/arcbees/bourseje/server/services/VoteService.java (hunk 2 above)
Comment (christiangoudreau) [inline src/main/java/com/arcbees/bourseje/server/services/VoteService.java]: Let's take a look at that after everything is being done as an improvement. The business case is not critical enough to give real business value for that improvement at the moment.

**Pull request merged**